config.go

```diff
@@ -177,12 +177,14 @@ func (c Configs) validateApps() error {
 func expansionFiles(appPaths []string, expansionFilePathConfig string) ([]string, error) {
 	// "main:/file/path/1.obb|patch:/file/path/2.obb"
 	expansionFileUpload := strings.TrimSpace(expansionFilePathConfig) != ""
-	expansionFileEntries := strings.Split(expansionFilePathConfig, "|")
-
-	if expansionFileUpload && (len(appPaths) != len(expansionFileEntries)) {
-		return []string{}, fmt.Errorf("mismatching number of APKs(%d) and Expansionfiles(%d)", len(appPaths), len(expansionFileEntries))
-	}
+	var expansionFileEntries = []string{}
 	if expansionFileUpload {
```

Comment:
Can just move `strings.TrimSpace(expansionFilePathConfig) != ""` to the if statement directly.

Reply:
Updated.